src/routes/products/[barcode]/ProductHeader.svelte

```
@@ -118,69 +203,83 @@
+							{:else if action.id === 'compare'}
+								<button
+									class="btn btn-secondary btn-sm md:btn-md"
+									onclick={addToComparison}
+									title={$_('product.buttons.compare')}
+									aria-label={$_('product.buttons.compare')}
+								>
+									<IconMdiCompare class="h-5 w-5" />
+								</button>
+							{/if}
+						{/if}
+					{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)